### PR TITLE
[READY] Add single tenant mode

### PR DESCRIPTION
**Pull request by fatboypunk**

### CODE REVIEW

File: app/models/concerns/devise_setup.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+               :invitable
+      else
+        devise :database_authenticatable,
+               :recoverable,
+               :rememberable,
+               :trackable,
+               :validatable,
+               :confirmable,
+               :invitable
+      end
```
Comment (jurre) [inline app/models/concerns/devise_setup.rb]: I don't really like this approach tbh, but if we go for this we could remove the duplication:
```
devise_options = [:database_authenticatable, :recoverable, :rememberable, :trackable, :validatable, :confirmable, :registerable, :invitable]
devise_options << :registerable if User.count == 0
devise *devise_options
```

Comment (fatboypunk) [inline app/models/concerns/devise_setup.rb]: I get that, I don't like it to create a default user. By changing the accounts/new form we also have to adjust the controller and the model of accounts to check on the single tenant mode

Comment (jurre) [inline app/models/concerns/devise_setup.rb]: Yeah makes sense. Let's just add a rake task that lets you set up the first user?

Comment (fatboypunk) [inline app/models/concerns/devise_setup.rb]: :+1:

**Comment by fatboypunk** [comment]
This is ready now @DefactoSoftware/hours

**Comment by jurre** [comment]
Sweet!

File: app/views/devise/registrations/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        = f.label :password
+        - if @validatable
+          %i
+            (#{@minimum_password_length} characters minimum)
+        = f.password_field :password, autocomplete: "off"
+      %div
+        = f.label :password_confirmation
+        = f.password_field :password_confirmation, autocomplete: "off"
+      %div= f.submit t("sign_up")
+    = render "devise/shared/links"
```
Comment (jurre) [inline app/views/devise/registrations/new.html.haml]: We don't need this anymore, right?

Comment (fatboypunk) [inline app/views/devise/registrations/new.html.haml]: Yeah forgot about this view, I'll remove it

**Comment by fatboypunk** [comment]
- [x] remove Sign up from locales
- [x] remove registration controller
- [x] update the readme to update the environment variables instead of just the .env
- [x] confirm the email of a user when created in a rake task

**Pull request merged**